pom.xml

```diff
@@ -566,6 +567,7 @@
                             </execution>
                         </executions>
                         <configuration>
+                            <doclint>none</doclint>
```

Comment:
What is the difference between this `doclint` parameter and the one used in the `additionalparam` parameter, in the line below this one?

Reply:
If you are running from maven with maven-javadoc-plugin v3.0.0 or later, you need to use the doclint setting.
When using earlier versions of maven-javadoc-plugin (before v3.0.0), you need to use a different setting: <additionalparam>-Xdoclint:none</additionalparam>